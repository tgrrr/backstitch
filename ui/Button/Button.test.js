import React from 'react'
import renderer from 'react-test-renderer'
import { Button } from '.'

it('Button: default', () => {
    const component = renderer.create(
        <Button>
            <>
                Button
            </>
        </Button>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
