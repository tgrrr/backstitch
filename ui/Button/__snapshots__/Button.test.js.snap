// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Button "
    data-testid="Button"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Button
      </span>
    </button>
  </div>
</div>
`;
